internal/policy/helpers_test.go

```diff
@@ -156,6 +156,11 @@ func createTestStateWithDelegatedPolicies(t *testing.T) *State {
 	}
 
 	key, err := tuf.LoadKeyFromBytes(rootPubKeyBytes)
+	if err != nil {
+		t.Fatal(err)
+	}
+	gpgkey, err := gpg.LoadGPGKeyFromBytes(gpgPubKeyBytes)
```

Comment:
We don't need this diff at all, right? Can you put gpgKey's load back where it was so this doesn't show up as a diff?

Reply:
Before it was tuf.LoadKeyFromBytes, which was failing, since it could not process a gpg key, so I had to switch it.

Reply:
https://github.com/gittuf/gittuf/pull/221/files#diff-ec90ffc9ec0e3e5ac57fce38f8a23d4e1b139204a779ac29bf13baa4a3c6dd6dL176-L180 already was loading this key, my point is we don't need this "move" anymore. By moving this back to the original location and fixing the var name, it'll have no diff at all.

Reply:
I have fixed this, but it is still showing a diff, even though I cannot find one.